Say Their Names/Source/Controller/Petitions/PetitionsViewCode.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  PetitionsUI.swift
+//  Say Their Names
+//
+//  Created by Kyle Lee on 6/2/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+/// The UI for Petitions
+final class PetitionsViewCode: UIView {
+    
+    private lazy var petitionsLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Petitions"
+        label.textAlignment = .center
+        
+        return label
+    }()
+    
+    init() {
```

Comment:
`init(frame: CGRect)` will work exactly the same, allowing to use both `init()` and `init(frame:)`. It's due to how initializers are inherited, can read more here (though it's a long read) : https://docs.swift.org/swift-book/LanguageGuide/Initialization.html